setup.py

```diff
@@ -281,7 +281,6 @@ def build_extension(self, ext):
     "snappy": ["python-snappy"],
     "zstd": ["zstandard"],
     "aws": ["pymongo-auth-aws<2.0.0"],
-    "srv": ["dnspython>=1.16.0,<3.0.0"],
```

Comment:
nb: if the user tries to use `pymongo[srv]`, they should see a warning that `srv` is not available but it should still work, from my experience with setuptools extras.

Reply:
e.g.:

```bash
Requirement already satisfied: pymongo[nonsense] in ./miniconda/lib/python3.10/site-packages (4.2.0)
WARNING: pymongo 4.2.0 does not provide the extra 'nonsense'
```

Reply:
Can we add this back in as "srv": [""] to suppress any warnings?

Reply:
According to https://setuptools.pypa.io/en/latest/userguide/dependency_management.html?highlight=extras#optional-dependencies this should be an empty list: `"srv": [],`

> Best practice: if a project ends up no longer needing any other packages to support a feature, it should keep an empty requirements list for that feature in its extras_require argument, so that packages depending on that feature don’t break (due to an invalid feature name).